lib/formatter/csv/annotation_for_csv.rb

```diff
@@ -14,7 +14,7 @@ def to_h
         case task['type']
         when "drawing"
           drawing
-        when "single", "multiple"
+        when /single|multiple|persistent/
```

Comment:
I'm not convinced about the persistent task type name here and am happy to change it.